specification/workloads/resource-manager/readme.python.md

```diff
@@ -17,3 +17,8 @@ clear-output-folder: true
 no-namespace-folders: true
 output-folder: $(python-sdks-folder)/workloads/azure-mgmt-workloads/azure/mgmt/workloads
 ```
+
+``` yaml $(python)
+modelerfour:
+  flatten-models: false
+```
```

Comment:
https://github.com/Azure/sdk-release-request/issues/4844 is not for workloads which is already GAed. I think it shall be https://github.com/Azure/azure-rest-api-specs/blob/main/specification/workloads/resource-manager/Microsoft.Workloads/SAPVirtualInstance/readme.python.md

Reply:
updated